packages/cli/test/mocks/team.ts

```diff
@@ -14,7 +15,7 @@ export function useTeams() {
   ];
 
   for (let team of teams) {
-    client.scenario.get(`/v1/team/${team.id}`, (_req, res) => {
+    client.scenario.get(`/teams/${team.id}`, (_req, res) => {
```

Comment:
The previous path `/v1/team` doesn't exist. I think this was written, but never invoked. The new path `/teams/:id` is invoked by the new tests.

Reply:
I'll defer to @TooTallNate here

Reply:
Ya, confirmed.